src/sentry/conf/server.py

```diff
@@ -2396,7 +2386,7 @@ def build_cdc_postgres_init_db_volume(settings):
 #
 # Currently, only redis is supported.
 SENTRY_REALTIME_METRICS_BACKEND = (
-    "sentry.processing.realtime_metrics.redis.RedisRealtimeMetricsStore"
+    "sentry.processing.realtime_metrics.dummy.DummyRealtimeMetricsStore"
 )
 SENTRY_REALTIME_METRICS_OPTIONS = {
```

Comment:
i'm sorely tempted to just set the options to `{}` as well, since they're not going to be used by a majority of users. thoughts?

Reply:
I prefer to leave them in, it does at least show how this is supposed to work.  The fact that the dummy backend ignores them is enough i think